package.json

```diff
@@ -3,6 +3,7 @@
   "description": "![logo](https://res.cloudinary.com/snyk/image/upload/v1468845259/logo/snyk-dog.svg) ## Snyk: NuGet Plugin ***",
   "main": "lib/index.js",
   "scripts": {
+    "build": "./node_modules/.bin/lodash -p -o ./dist/lodash-min.js include=uniq",
```

Comment:
not sure about that ...

Reply:
we did it in the php plugin, but no excuse ;)
what do you suggest?

Reply:
You did something different on the php plugin - the `dist` folder is built on travis and pushed to npmjs, not checked in to git.
Not an issue really because we won't need this once the duplication issue is resolved?